lighthouse-core/gather/gather-runner.js

```diff
@@ -225,20 +225,20 @@ class GatherRunner {
 
   /**
    * Returns an error if we try to load a non-HTML page.
```

Comment:
come to think of it... any reason for not just moving the redirect resolution into here? :)

on the other hand, we might end up needing the resolved request elsewhere too and the visibility might be nice to have to future readers of the distinction. if we keep it outside of this fn, let's add an extra a little comment that it expects redirects resolved.
```suggestion
   * Returns an error if we try to load a non-HTML page.
   * Expects a network request with all redirects resolved, otherwise the MIME type may be incorrect.
```

Reply:
I think the reasoning makes sense to keep it separate, but whichever you or @Beytoven prefer! I'll add a comment for now.

Reply:
I like them being separate as well so i think we're good here. Thanks for adding the comment.